packages/11ty/_layouts/search-index.11ty.js

```diff
@@ -5,7 +5,9 @@ module.exports = function ({ pages }) {
         abstract: page.data.abstract,
         contributor: page.data.contributor,
         content: page.data.content,
-        length: page.data.content.split(' ').length, // @todo replace with a real wordcount if wordcount is necessary
+        length: page.data.content
+          ? page.data.content.split(' ').length
+          : 0, // @todo replace with a real wordcount if wordcount is necessary
```

Comment:
I had to add this ternary because of build errors due to `content` being undefined. This is worth re-implementing in a more solid way; fixing this seemed beyond the scope of this already large task